src/components/network/NetworkItem.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="network-item">
```

Comment:
reminder: we discussed some indicator for which network is selected. checkmark or pink outline would suffice.

Reply:
Done